src/main/java/org/opensearch/performanceanalyzer/listener/RTFPerformanceAnalyzerSearchListener.java

```diff
@@ -38,6 +39,7 @@ public class RTFPerformanceAnalyzerSearchListener
             LogManager.getLogger(RTFPerformanceAnalyzerSearchListener.class);
     private static final String SHARD_FETCH_PHASE = "shard_fetch";
```

Comment:
Please change to Otel standards shard.fetch this is same for all dimensions added